source/codegen/metadata/nidmm/config.py

```diff
@@ -1,29 +1,16 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-DMM API metadata version 19.6.0d7
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 22.8.0d9999
 config = {
     'additional_headers': {
-        'custom/ivi_errors.h': [
```

Comment:
There aren't any errors in the build without this included by the service.cpp so I think it's fine to remove.